src/activities/Elsa.Activities.Timers.Hangfire/Extensions/TimersOptionsExtensions.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+
+using Elsa.Activities.Timers.Hangfire.Services;
+using Elsa.Activities.Timers.Options;
+using Elsa.Activities.Timers.Services;
+
+using Hangfire;
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Elsa.Activities.Timers.Hangfire.Extensions
+{
+    public static class TimersOptionsExtensions
+    {
+        /// <summary>
+        /// Add Hangfire for background processing
+        /// </summary>
+        /// <param name="timersOptions"></param>
+        public static void UseHangfireProviderCore(this TimersOptions timersOptions)
+        {
+            timersOptions.Services
+                .AddSingleton<IWorkflowScheduler, HangfireWorkflowScheduler>()
+                .AddSingleton<ICrontabParser, HangfireCrontabParser>();
+        }
+
+        /// <summary>
+        /// Add Hangfire for background processing
+        /// </summary>
+        /// <remarks>
+        /// Only if Hangfire is not already registered in DI
+        /// </remarks>
+        /// <param name="timersOptions"></param>
+        /// <param name="timersOptions">Hangfire settings</param>
+        public static void UseHangfireProvider(this TimersOptions timersOptions, Action<IGlobalConfiguration> configure)
```

Comment:
Perhaps merge this method with `UseHangfireProviderCore` and make `configure` nullable and optional? At the very least, both methods should have the same name.

Reply:
I did it this way in case someone would prefer to add Hangfire directly to their application instead of through Elsa. I for example use it directly in my web application and configure it in the startup.

I didn't necessarily want to add it again via Elsa. Or let's say if `configure` is null, then only the services are added.

Reply:
I see. That makes sense. Leave it as-is.